bundle/edu.gemini.spdb.reports.collection/src/main/scala/edu/gemini/spdb/cron/osgi/Activator.scala

```diff
@@ -37,7 +38,8 @@ class Activator extends BundleActivator {
          "odbState"       -> OdbStateAgent.run,
          "odbMail"        -> OdbMailAgent.run,
          "weather"        -> new WeatherUpdater(c).run,
-         "archive"        -> Archiver.run(c))
+         "archive"        -> Archiver.run(c),
```

Comment:
How do you define the cron frequency? Or is it that an external cronjob makes a http request that triggers the task?